sqlmesh/dbt/basemodel.py

```diff
@@ -140,8 +142,20 @@ def _validate_columns(cls, v: t.Any) -> t.Dict[str, ColumnConfig]:
 
     @property
     def all_sql(self) -> SqlStr:
+        return SqlStr("\n".join([self.hook_sql, self.sql_no_config]))
+
+    @property
+    def sql_no_config(self) -> SqlStr:
+        return SqlStr("")
+
+    @property
+    def sql_config(self) -> SqlStr:
```

Comment:
Is this sql with config or just the config portion? If the former perhaps it's worth calling `sql_with_config`.

Reply:
Its the latter, but since the name is confusing, I changed to `sql_embedded_config`.